### PR TITLE
chore: remove unnecessary TODOs from api, update TODOs with issue numbers

### PR DESCRIPTION
**Pull request by tenthirtyone**
Housekeeping to update code TODOs with links to matching Issues.

### CODE REVIEW

File: src/chains/ethereum/ethereum/src/data-managers/blocklog-manager.ts [hunk 1]
```diff
@@ -65,6 +65,8 @@ export default class BlockLogManager extends Manager<BlockLogs> {
         blockLogsRange.forEach(blockLogs => {
           // TODO(perf): this loops over all addresses for every block.
           // Maybe make it loop only once?
+          // Issue: https://github.com/trufflesuite/ganache/issues/3482
+
```
Comment (MicaiahReid) [inline src/chains/ethereum/ethereum/src/data-managers/blocklog-manager.ts]:
```suggestion
```

**Review by MicaiahReid — CHANGES_REQUESTED** [review]
I think I'd probably consider this a `chore` PR rather than a `fix`, but some might disagree. 

Can the description be updated to be release ready?

**Review by jeffsmale90 — APPROVED** [review]

File: src/chains/ethereum/ethereum/src/data-managers/blocklog-manager.ts (hunk 1 above)
Comment (tenthirtyone) [reply src/chains/ethereum/ethereum/src/data-managers/blocklog-manager.ts]: :+1:

**Review by MicaiahReid — APPROVED** [review]

**Review by davidmurdoch — APPROVED** [review]

**Comment by davidmurdoch** [comment]
I've updated the PR title to reflect the plural changes of the PR. When merging this can make sure sure the commit message reflects this change?

**Pull request merged**